recipe/meta.yaml

```diff
@@ -32,6 +31,17 @@ requirements:
     - pytest-runner
     - python
     - pytorch >=1.7
+    - numpy
+    - tqdm
+    - scipy
+    - networkx
+    - scikit-learn
+    - requests
+    - pandas
+    - rdflib
+    - googledrivedownloader
+    - jinja2
+    - pyparsing
   run:
     - python
     - pytorch >=1.7
```

Comment:
Try this:

```suggestion
    - {{ pin_compatible('pytorch', max_pin='x.x.x') }}
```

Reply:
This shouldn't be needed as of 1.9.0, though :/

Reply:
This is correct, you should likely just leave it blank.

Reply:
Is this fine? https://github.com/sarthakpati/torch-scatter-feedstock/blob/full_build/recipe/meta.yaml#L47

Reply:
Yep, that looks fine. Let's see how it goes!

Reply:
Thanks @hmaarrfk!